src/modules/10-common/components/MultiTypeExecutionCondition/MultiTypeExecutionCondition.tsx

```diff
@@ -17,7 +17,7 @@ import css from './MultiTypeExecutionCondition.module.scss'
 interface MultiTypeExecutionConditionProps {
   path: string
   allowableTypes?: MultiTypeInputType[]
-  formik?: FormikContext<any>
+  formik?: FormikContextType<any>
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232293)
```suggestion
  formik?: FormikContextType<unknown>
```